documentation/sequence-diagram-authorization.md

```diff
@@ -0,0 +1,21 @@
+```mermaid
+sequenceDiagram
+  participant W as Web App
+  participant O as Okta
```

Comment:
I think if you swap server and Okta in this list the diagram looks better.  You can render this diagram here to see  https://mermaid-js.github.io/mermaid-live-editor/#/edit/eyJjb2RlIjoic2VxdWVuY2VEaWFncmFtXG4gIHBhcnRpY2lwYW50IFcgYXMgV2ViIEFwcFxuICBwYXJ0aWNpcGFudCBTIGFzIFNlcnZlclxuICBwYXJ0aWNpcGFudCBPIGFzIE9rdGFcblxuICBXLT4-UzogR2V0IHVzZXIgaW5mb3JtYXRpb25cbiAgTm90ZSBvdmVyIFcsUzogR0VUIC9tZSB3aXRoIGFjY2VzcyB0b2tlbiBhcyBCZWFyZXJcbiAgUy0tPj5POiBWYWxpZGF0ZSB1c2VyIGFjY2VzcyB0b2tlblxuICBOb3RlIG92ZXIgTyxTOiB2ZXJpZnkudmVyaWZ5QWNjZXNzVG9rZW4oPGJyLz50b2tlbiwgYXBwbGljYXRpb25OYW1lPGJyLz4pXG4gIE8tPj5TOiBSZXR1cm4gT2t0YSBhY2Nlc3MgY2xhaW1zXG4gIE5vdGUgb3ZlciBPLFM6IHsgLi4uLCB1aWQ6IHVzZXJJZCwgLi4uIH1cbiAgUy0tPj5POiBHZXQgdXNlciBwcm9maWxlXG4gIE5vdGUgb3ZlciBPLFM6IG9rdGFDbGllbnQuZ2V0VXNlcih1c2VySWQpXG4gIE8tPj5TOiBSZXR1cm4gdXNlciBwcm9maWxlXG4gIE5vdGUgb3ZlciBPLFM6IHsgPGJyIC8-Li4uLDxici8-dWlkOiB1c2VySWQsPGJyIC8-ZGlzcGxheU5hbWU6IGRpc3BsYXlOYW1lLDxiciAvPnN0YXR1czogJ0FDVElWRScsPGJyIC8-Li4uIDxiciAvPn1cbiAgUy0-PlM6IFJldHJpZXZlIGF1dGhvcml6YXRlZCByb2xlcyBmcm9tIGRhdGFiYXNlXG4gIFMtPj5TOiBSZXRyaXZlIGFsbG93ZWQgYWN0aXZpdGllcyBmcm9tIGRhdGFiYXNlXG4gIFMtLT4-VzogUmV0dXJuIHVzZXIgcHJvZmlsZSwgcm9sZSwgYW5kIGFjY2Vzc1xuICBOb3RlIG92ZXIgVyxTOiB7PGJyLz4uLi4sPGJyPmFjdGl2aXRpZXM6IFthY3Rpdml0aWVzXSw8YnIvPmlkOiB1c2VySWQsPGJyLz4gbmFtZTogZGlzcGxheU5hbWUsPGJyLz5zdGF0ZTogc3RhdGUsPGJyLz5yb2xlOiByb2xlLDxici8-Li4uPGJyLz59IiwibWVybWFpZCI6e30sInVwZGF0ZUVkaXRvciI6ZmFsc2V9

Reply:
The frontend communicates directly with Okta though and then once the person is authenticated they communicate with our server. Our server does communicate with Okta too, but that's only for authorization not authentication

Reply:
Disregard previous, yes, we can change this. :D